packages/charts/src/vaadin-chart-mixin.d.ts

```diff
@@ -207,6 +216,14 @@ export type ChartYaxesExtremesSetEvent = CustomEvent<{
   };
 }>;
 
+export type ChartLegendItemClickEvent = CustomEvent<{
```

Comment:
Let's add a JSDoc comment `Fired when the legend item is clicked.` for consistency with other types.